examples/pytorch/graphsage/dist/train_dist_transductive.py

```diff
@@ -427,7 +334,8 @@ def main(args):
     parser.add_argument(
         "--part_config", type=str, help="The path to the partition config file"
     )
-    parser.add_argument("--num_clients", type=int, help="The number of clients")
+    parser.add_argument(
+        "--num_clients", type=int, help="The number of clients")
```

Comment:
same comment

Reply:
ditto, not used at all. let me remove it.